### PR TITLE
Map moss

### PR DESCRIPTION
**Pull request by haykam821**
This pull request maps the moss carpets and moss blocks added in 21w05a.

### CODE REVIEW

**Comment by OroArmor** [comment]
Shouldnt this also include the moss blocks in `Blocks.java`? I only see the items for the new moss blocks

**Comment by haykam821** [comment]
That isn't necessary since they are [auto-mapped](https://github.com/FabricMC/yarn/blob/323e6f8e29e7064360fecb0037e076c5e073dc72/mappings/net/minecraft/block/Blocks.mapping) based on [the string passed to the register method](https://github.com/FabricMC/yarn/blob/323e6f8e29e7064360fecb0037e076c5e073dc72/mappings/net/minecraft/block/Blocks.mapping#L147).

**Comment by OroArmor** [comment]
oh wait, yarn is way smarter than i thought

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/item/Items.mapping [hunk 1]
```diff
@@ -241,6 +241,8 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_28043 DRIPSTONE_BLOCK Lnet/minecraft/class_1792;
 	FIELD field_28101 SCULK_SENSOR Lnet/minecraft/class_1792;
 	FIELD field_28409 GLOW_LICHEN Lnet/minecraft/class_1792;
+	FIELD field_28653 MOSS_CARPET Lnet/minecraft/class_1792;
+	FIELD field_28654 MOSS_BLOCK Lnet/minecraft/class_1792;
```
Comment (liach) [inline mappings/net/minecraft/item/Items.mapping:245]: Is the item id `moss_block` and the block it `moss`? If yes guess it's just another lilypad or cauldron

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/item/Items.mapping (hunk 1 above)
Comment (haykam821) [reply mappings/net/minecraft/item/Items.mapping:245]: No, the block and item identifiers are the same. The sound events just use `moss`, though.

Comment (liach) [reply mappings/net/minecraft/item/Items.mapping:245]: Hmm, If they are both `moss_block` I'd suggest naming the block field `MOSS_BLOCK` as well, given how `GOLD_BLOCK` etc are named

Comment (haykam821) [reply mappings/net/minecraft/item/Items.mapping:245]: Hm? It is already auto-mapped as `Blocks#MOSS_BLOCK`; the `MOSS` in this pull request is a `BlockSoundGroup` containing sound events such as `block.moss.break`.

**Review by jaskarth — APPROVED** [review]

**Pull request merged**